bootstrap_cfn/config.py

```diff
@@ -740,6 +743,17 @@ def elb(self, template):
                 ),
                 Policies=elb_policies
             )
+
+            if elb['scheme'] == 'internet-facing':
+                vpc_gw_att = [
+                    r.title
+                    for r in self._find_resources(
+                        template, "AWS::EC2::VPCGatewayAttachment")]
```

Comment:
With the previous comment in mind, this could also make use of the same variable.